politeiawww/www.go

```diff
@@ -1255,6 +1255,13 @@ func (p *politeiawww) handleEditProposal(w http.ResponseWriter, r *http.Request)
 	util.RespondWithJSON(w, http.StatusOK, epr)
 }
 
+// handleNotFound is a generic handler for an invalid route.
```

Comment:
Copy paste error, comment is wrong for the method

Reply:
good catch, fixing it